### PR TITLE
chore: update every tests to run mode

### PR DESCRIPTION
**Pull request by jakovljevic-mladen**
Updated `every` tests to use run mode.

### CODE REVIEW

**Comment by cartant** [comment]
I saw the PR title and thought you'd updated every operator in a single PR 😲😂

File: spec/operators/every-spec.ts [hunk 1]
```diff
@@ -1,274 +1,365 @@
...
 
-    expectObservable(source.pipe(every(predicate))).toBe(expected, {x: true});
-    expectSubscriptions(source.subscriptions).toBe(sourceSubs);
+      expectObservable(e1.pipe(every(predicate))).toBe(expected, { x: true });
+      expectSubscriptions(e1.subscriptions).toBe(e1subs);
+    });
   });
 
-  it('should emit true if Scalar source matches with predicate', () => {
-    const source = of(5);
```
Comment (cartant) [inline spec/operators/every-spec.ts:160]: issue: Probably best to not change any sources that are currently declared using `of()` to `cold()`, as there is a subtle difference - see #5760 and its associated issue for background.

Is this something that you've done in other PRs? I'm wondering whether or not I missed it in any of the other reviews.

**Review by cartant — CHANGES_REQUESTED** [review]
This is all okay, but I don't think we should be changing `of()` to `cold()`. See my comment below.

File: spec/operators/every-spec.ts (hunk 1 above)
Comment (jakovljevic-mladen) [reply spec/operators/every-spec.ts:160]: OK, I will change this, but I have been changing this in my other PRs as well. Should I revert from other tests?

Comment (jakovljevic-mladen) [reply spec/operators/every-spec.ts:160]: Oh, didn't see that you already asked... Yes, I was changing in other PRs as well. I'll make a revert PR.

Comment (jakovljevic-mladen) [reply spec/operators/every-spec.ts:160]: @cartant I started reading #5760 and I found it to be quite large to me. I'll give it try next time, but for now, out of curiosity, I'd like to know what would e.g. `cold('a')` represent? This:
`new Observable(subscriber => subscriber.next('a'))`
or this:
`new Observable(subscriber => setTimeout(() => subscriber.next('a'), 0))`
Is #5760 trying change behavior from one to another maybe?

Comment (cartant) [reply spec/operators/every-spec.ts:160]: `cold('a')` essentially represents `new Observable(subscriber => asapScheduler.schedule(() => subscriber.next('a')))`.

Comment (jakovljevic-mladen) [reply spec/operators/every-spec.ts:160]: Updated PR, I think I reverted everything. Other reverts coming...
I didn't know about this subtle distinction. Thanks a lot for explanations Nicholas. Any chance to explain why `asapScheduler` is used for completion notifications? If easier, you can post a link to an issue or to a source code.
Is completion notification scheduled on `asapScheduler` only for frame 0 or for every frame?

Comment (cartant) [reply spec/operators/every-spec.ts:160]: Thanks for the update.

Actually, what I said above was wrong. [All notifications are emitted on the test scheduler](https://github.com/ReactiveX/rxjs/blob/1f021fe280e707a6bfbd36dcc8880d17c3e64fd2/src/internal/testing/ColdObservable.ts#L35-L50), so it's kinda equivalent to:

```ts
new Observable(subscriber => {
  asapScheduler.schedule(() => subscriber.next('a'));
  asapScheduler.schedule(() => subscriber.complete());
})
```

**Review by cartant — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**